distribution/pom.xml

```diff
@@ -95,6 +95,7 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <configuration>
+                    <additionalJOption>-addmods java.xml.bind</additionalJOption>
```

Comment:
This breaks the build on Java 8. It should be added in a profile (or removed as you claim we do not support building the distribution with Java 9).

Thanks!